index.js

```diff
@@ -13,9 +13,12 @@ const config = require('./config');
 config.init();
 const plugins = require('./plugins');
 
+const windowSet = new Set([]);
+
 // expose to plugins
 app.config = config;
 app.plugins = plugins;
+app.getWindows = () => [...windowSet];
```

Comment:
I do. If they remove/add/modify something from `Set` without cloning it, it could cause issues. We could create a _new_ `Set` (i.e. clone it) and return that if you'd like.